USAGE.md

```diff
@@ -122,6 +121,26 @@ To address this, you may:
 psql --dbname=mds --username=mds --host=localhost --password --port=5432
 ```
 
+### Using the Document Manager and Document Generator locally
+
+If you are running the frontend using npm run serve then you will not be able to use the document manager at the same time as the document generator. If you wish to do this then you need to make an addition to your hosts file so the browser can resolve the document_manager_backend to localhost.
+
+If you are on a windows machine or using power shell run the folowing command at the root of this project:
```

Comment:
Maybe edit the note to say that it needs to run in administrator mode in the power shell:
![image](https://user-images.githubusercontent.com/17113094/121402391-b054ba80-c90e-11eb-8450-70a002408ed7.png)